src/commands/cmd_query.c

```diff
@@ -187,7 +187,7 @@ static void _ExecuteQuery(void *args) {
 
 	// instantiate the query ResultSet
 	bool compact = command_ctx->compact;
-	ResultSetFormatterType resultset_format = profile
+	ResultSetFormatterType resultset_format = profile || command_ctx->replicated_command
```

Comment:
I think a comment is in order to explain why we're using FORMATTER_NOP in case `command_ctx->replicated_command`